tests/boards/ehl_crb/gpio/sample.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+sample:
+  description: GPIO sample app.
+  name: gpio
+  platforms: ehl_crb
```

Comment:
I suspect the reason CI fails is that this should be called `platforms_allow` (not 100% sure though)